website/client/js/controllers/partyCtrl.js

```diff
@@ -183,6 +186,7 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
       $scope.questForceStart = function(){
         Quests.sendAction('quests/force-start')
           .then(function(quest) {
+            console.log(quest)
```

Comment:
I've removed it when I merged
Il 18 mag 2016 1:52 PM, "Blade Barringer" <notifications@github.com> ha
scritto:

> In website/client/js/controllers/partyCtrl.js
> <https://github.com/HabitRPG/habitrpg/pull/7341#discussion_r63689021>:
>
> > @@ -183,6 +186,7 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
> >        $scope.questForceStart = function(){
> >          Quests.sendAction('quests/force-start')
> >            .then(function(quest) {
> > +            console.log(quest)
>
> console.log
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7341/files/ba5745cb376132a46f8d752c1c9d833d53271675#r63689021>
>